### ISSUE TITLE
New DL Client Health check throws false negative

### ISSUE THREAD

**Issue by stevietv** [Sonarr/Sonarr#5522]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

the new healthcheck on dl client folder paths brought in by @bakerboy448 in #5462 via commit 0f6f6814388b25b8fb3fccb6f33bfa16e962549f can throw a false negative:

![image](https://user-images.githubusercontent.com/1117625/229030633-94861754-007d-407f-8e28-f0034776d35c.png)

This happens when the standard download location is set to a different folder than the category specific downloads location that is absolute and not relative to the default save path.

![image](https://user-images.githubusercontent.com/1117625/229030899-19efa091-90b3-4b09-9f3b-f15b21ace8f1.png)

![image](https://user-images.githubusercontent.com/1117625/229030930-98d307f0-84db-40a6-9d50-1e091f16bdf7.png)

### Expected Behavior

Check should be made against category specific download location

### Environment

```markdown
- OS: n/a
- Sonarr: 4.0.0.418 upwards
- Docker Install: yes
- Using Reverse Proxy: n/a
- Browser: n/a
```

### What branch are you running?

Develop

### Trace Logs?

n/a

**Comment by bakerboy448** [comment]
Looks like this is due to the qbit get status only taking into account the config save path

https://github.com/Sonarr/Sonarr/blob/a776b68574bbf9315f3d18e55e0f310e95a37e79/src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs#L365

**Comment by raidenlal** [comment]
I have similar issue with qbittorrent being on host machine and sonarr inside docker - both see downloads folder, but qbittorrent sees it as native host application as "c:\torrentdownloads" and sonarr sees it as "/dowloads" which is mapped to the same host directory as qbittorrent uses.

Is this the same issue or should I open new one?

Thank you

**Comment by stevietv** [comment]
@raidenlal - you should probably first come to one of our support channels first: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC](https://web.libera.chat/?channels=#sonarr) for assistance.

**Comment by raidenlal** [comment]
@stevietv not really sure why? It is just a false positive check, everything is working, just this notification started to pop-up in last few weeks

**Comment by stevietv** [comment]
@raidenlal - sure, then wait and see when #5523 is merged if it removes the erroneous error
